package/kedro_viz/server.py

```diff
@@ -45,13 +45,21 @@
 import requests
 from flask import Flask, jsonify, send_from_directory
 from IPython.core.display import HTML, display
-from kedro.cli import get_project_context  # pylint: disable=ungrouped-imports
-from kedro.cli.utils import KedroCliError  # pylint: disable=ungrouped-imports
-from semver import match
+from semver import VersionInfo
 from toposort import toposort_flatten
 
 from kedro_viz.utils import wait_for
 
+KEDRO_VERSION = VersionInfo.parse(kedro.__version__)
+
+if KEDRO_VERSION.match(">=0.16.0"):
```

Comment:
I think at some point we should stop doing this and bump supported Kedro version in Viz. Otherwise it rapidly becomes an `if/else` spaghetti, imo.

Reply:
Yep agreed. We have support for 0.14 in there too, that should be the first to go.